spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -58,7 +69,7 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-kafka</artifactId>
-			<version>${spring-integration-kafka.version}</version>
+			<version>2.1.1.BUILD-SNAPSHOT</version>
```

Comment:
Looks like `<spring-integration-kafka.version>2.1.0.RELEASE</spring-integration-kafka.version>` can be used from the root POM. 
Why do you do that explicitly here?
Thanks